compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/gating-preserves-function-properties.tsx

```diff
@@ -1,10 +1,5 @@
 // @gating
 
-/**
- * Fail: bug-gating-invalid-function-properties
- *   Unexpected error in Forget runner
- *   Component is not defined
- */
```

Comment:
note: also update SproutTodoFilter to remove "bug-gating-invalid-function-properties"